src/frontend/src/components/FillForms/index.tsx

```diff
@@ -1,111 +1,112 @@
-import React from "react";
+import React, { useEffect } from "react";
 import UserDataForm from "./UserDataForm";
 import { HashLink as Link } from "react-router-hash-link";
 
-interface Props {}
+// interface Props {}
 
-interface State {}
+// interface State {}
 
-export default class FillForms extends React.Component<Props, State> {
+// export default class FillForms extends React.Component<Props, State> {
+// componentDidMount() {
+//   document.title = "Generate Expungement Forms - RecordSponge";
+// }
 
-  componentDidMount(){
+export default function FillFormsIndex() {
+  useEffect(() => {
     document.title = "Generate Expungement Forms - RecordSponge";
-  }
+  }, []);
 
-  public render() {
-    return (
-      <main className="mw8 center f6 f5-l ph3 pt4 pb6">
-        <h1 className="f3 f2-l fw9 mb4">Generate Expungement Forms</h1>
-        <div className="flex-l pt2">
-          <div className="w-50-l pr4-l mb4">
-            <section className="mw6 lh-copy">
-              <p className="mb3">
-                This will fill and download the required paperwork forms as PDF
-                files for all cases that have charges eligible for expungement.
-              </p>
+  return (
+    <main className="mw8 center f6 f5-l ph3 pt4 pb6">
+      <h1 className="f3 f2-l fw9 mb4">Generate Expungement Forms</h1>
+      <div className="flex-l pt2">
+        <div className="w-50-l pr4-l mb4">
+          <section className="mw6 lh-copy">
+            <p className="mb3">
+              This will fill and download the required paperwork forms as PDF
+              files for all cases that have charges eligible for expungement.
+            </p>
 
-              <p className="mb3">
-                It includes cases that have an outstanding balance. These
-                balances must be paid off before filing the paperwork. Consult
-                the section on{" "}
-                <Link
-                  className="link bb hover-dark-blue"
-                  to="/manual#paybalances"
-                >
-                  paying balances
-                </Link>{" "}
-                in the Manual.
-              </p>
+            {/* <p className="mb3">
+                  It includes cases that have an outstanding balance. These
+                  balances must be paid off before filing the paperwork. Consult
+                  the section on{" "}
+                  <Link
+                    className="link bb hover-dark-blue"
+                    to="/manual#paybalances"
+                  >
+                    paying balances
+                  </Link>{" "}
+                  in the Manual.
+                </p> */}
```

Comment:
Do we no longer think this is relevant because we don't allow these cases by default? If so, I prefer to just remove.